src/spork.cpp

```diff
@@ -74,6 +74,7 @@ void CSporkManager::LoadSporksFromDB()
         }
 
         // add spork to memory
+        AddSporkMessage(spork);
         mapSporks[spork.GetHash()] = spork;
         mapSporksActive[spork.nSporkID] = spork;
```

Comment:
You missed to remove this two lines, `AddSporkMessage` is already adding/updating the map values.

edit: you fixed it in the next commit.